lua/nvim_lsp/configs.lua

```diff
@@ -33,26 +33,26 @@ function configs.__newindex(t, config_name, config_def)
   -- The config here is the one which will be instantiated for the new server,
   -- which is why this is a function, so that it can refer to the settings
   -- object on the server.
-  local function add_callbacks(config)
-    config.callbacks["window/logMessage"] = function(err, method, params, client_id)
+  local function add_handlers(config)
+    config.handlers["window/logMessage"] = function(err, method, params, client_id)
       if params and params.type <= config.log_level then
         -- TODO(ashkan) remove this after things have settled.
-        assert(lsp.callbacks, "Update to Nvim HEAD. This is an incompatible interface.")
-        assert(lsp.callbacks["window/logMessage"], "Callback for window/logMessage notification is not defined")
-        lsp.callbacks["window/logMessage"](err, method, params, client_id)
+        assert(lsp.handlers, "Update to Nvim HEAD. This is an incompatible interface.")
```

Comment:
I think we should `assert(not lsp.callbacks, ...` with an informative message and a help section to look at .

Once we do that, we can merge. ("Update to Nvim HEAD" doesn't tell people what / how they need to fix anything)

Reply:
Did the most recent commit address this well?